tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -305,63 +345,86 @@ def test_matchmaker_add(app, institute_obj, case_obj):
         assert resp.status_code == 302
 
 
-def test_matchmaker_matches(app, institute_obj, case_obj, mme_submission):
```

Comment:
Ehrm, did you just change the order of these two? No, there is some reworking going on here, that is not exactly related to the introduction of age or tissue. Looks ok, but could we just get a brief summary of that too!

Reply:
Oh yes, I've improved the test so it's covering more code of the matchmaker_matches function. Not related at all with the main topic of this PR!